includes/init.php

```diff
@@ -64,6 +64,7 @@ function module_selected($module, $modules)
 require_once $install_dir . '/includes/datastore.inc.php';
 require_once $install_dir . '/includes/billing.php';
 require_once $install_dir . '/includes/syslog.php';
+require_once $install_dir . '/includes/snmptrap.inc.php';
```

Comment:
We shouldn't be including this for all scripts, it should be just included in snmptrap.php.

I'm going to bump this to 1.43 release.

Reply:
Honestly, it is just one function.  I would just put the function in one of the existing includes.

Reply:
I can either move the function to functions.php or the include to snmptrap.php

Reply:
I think that the laf option is better. Why need all (gui/cli) load the trap function (even if its only 1) if only snmptrap.php is going to use it? Also, dont know if for doing test you need to load the function in the test so Im lost.

Will do whatever you guys decide

Reply:
I really wish I could but thats really out of my skill. If you point me with good examples of function to classes maybe, with time, I can get it working in a class.
For now, I'm going to move the include to snmptrap.php.